coalib/bearlib/languages/documentation/DocumentationComment.py

```diff
@@ -12,6 +12,7 @@ class DocumentationComment:
     inside source-code, like position etc.
     """
     Parameter = namedtuple('Parameter', 'name, desc')
+    ExceptionValue = namedtuple('ExceptionValue', 'name, desc')
```

Comment:
`Exception` is a class in python. It is used as the base class for all exceptions. I guess i cant re-name it to that :stuck_out_tongue:  
https://www.tutorialspoint.com/python/python_exceptions.htm